src/Map.js

```diff
@@ -177,9 +177,36 @@ export default class Map {
         () => this.goPrevPoint(),
       )
       this.buttonPanel.mount(mountButtonPanelTarget)
-      this.on(Map.REGISTERED_EVENTS.TREE_SELECTED, () =>
-        this.buttonPanel.show(),
-      )
+      this.on(Map.REGISTERED_EVENTS.TREE_SELECTED, () => {
+        const currentPoint = this.layerSelected.payload
+        const points = this.getPoints()
+        const index = points.reduce((a, c, i) => {
+          if (c.id === currentPoint.id) {
+            return i
+          }
+          return a
+        }, -1)
+        if (points.length <= 1) {
+          return null
+        }
+        this.buttonPanel.show()
+        if (index === 0) {
+          document.querySelector('#next-left-arrow > svg').style.display =
```

Comment:
For the styling of buttons, instead of writing the code (line 194-207) inside Map.js, we could perhaps move these to the ButtonPanel.js and create three different methods like hideLeftArrow(), hideRightArrow(), and showBothArrows(). We can then access them inside Map.js as:
```
if (index === 0) {
   this.this.buttonPanel.hideLeftArrow()
}
```

Reply:
@jamwalab thanks for your great job, it looks great, and I think @pratikmdhr is right, the goal of the ButtonPanel is to isolate the feature of panel to component.  So please move these code into it, keep in mind, the Map.js just know buttonpanel object, shouldn't have any knowledge like: `#next-right-arrow`

Reply:
Sorry for the late reply, was out on vacation. Will work on this :)

Reply:
no worries!